README.md

```diff
@@ -1,70 +1,65 @@
-# Getting Started with Create React App
+# Describtion
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Corona Statisitics.
 
-## Available Scripts
+# Video Presentation
+https://www.youtube.com/watch?v=pqE2OsOnVpY
```

Comment:
- [ ] In the video you are expected to highlight a piece of code you found interesting, please record an additional video touching on this point.